src/main/java/org/terasology/structureTemplates/util/BlockRegionUtilities.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.structureTemplates.util;
+
+
+import org.terasology.math.Region3i;
+import org.terasology.math.geom.Vector3f;
+import org.terasology.math.geom.Vector3i;
+import org.terasology.structureTemplates.components.SpawnBlockRegionsComponent;
+import org.terasology.structureTemplates.util.transform.BlockRegionMovement;
+
+import java.util.List;
+
+
+/**
+ * A collection of little helper to ease the work with structure templates.
+ **/
+
+public class BlockRegionUtilities {
+
+    public static BlockRegionMovement setOnCenterXZ(SpawnBlockRegionsComponent spawnBlockRegionsComponent) {
```

Comment:
I still would go for an even more general version, like
```Java
public static Vector3i determineCenterPointAlongAxisOf(SpawnBlockRegionsComponent component, Axis... axis)
```
What do you think? `determineBottomCenterPointOf` could be implemented as:
```Java
public static Vector3i determineBottomCenterPointOf(SpawnBlockRegionsComponent component) {
    return determineCenterPointAlongAxisFor(component, Axis.X, Axis.Z);
}
```

P.S.: I don't think we have general axis descriptors yet, only [`Direction`](https://github.com/MovingBlocks/Terasology/blob/develop/engine/src/main/java/org/terasology/math/Direction.java).